src/test/resources/test_data/calculation-service-examples.csv

```diff
@@ -135,12 +132,6 @@ custom-examples,crs-816-ac3,
 custom-examples,crs-816-ac4,
 custom-examples,crs-816-extra-ac,
 custom-examples,crs-822-tagged-bail-recall-bug,
-custom-examples,crs-657-concurrent-different-date,CustodialPeriodExtinguishedException
```

Comment:
These 'post validation' error scenarios run after the calculation and have been abstracted out to not in the calculate method. TODO move the tests